core/src/main/scala/com/karumi/shot/Shot.scala

```diff
@@ -109,12 +109,23 @@ class Shot(adb: Adb,
       comparison,
       newScreenshotsVerificationReportFolder,
       shouldPrintBase64Error)
-    screenshotsSaver.copyRecordedScreenshotsToTheReportFolder(
-      projectFolder,
-      flavor,
-      buildType,
-      buildFolder + Config
-        .verificationReportFolder(flavor, buildType) + "/images/recorded/")
+
+    if (showOnlyFailingTestsInReports) {
+      screenshotsSaver.copyOnlyFailingRecordedScreenshotsToTheReportFolder(
```

Comment:
Instead of creating a new method could we just add the ``showOnlyFailingTestsInReports`` as a parameter of the existing one?

Reply:
Sure, I can combine it.

Reply:
Thanks!

Reply:
This does not work as ScreenshotsSaver.copyRecordedScreenshotsToTheReportFolder(…) is also used in other places where updadedComparison is not available, e.g. in Shot.recordRegularScreenshots(…).

Or am I missing something?